CONTRIBUTING.md

```diff
@@ -100,3 +127,34 @@ With your PR raised, and the CI showing green, your PR will [sit in the queue to
+
+## Financial contributions
+
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/typescript-eslint).
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+### Contributors
+
+Thank you to all the people who have already contributed to typescript-eslint!
+<a href="https://github.com/typescript-eslint/typescript-eslint/graphs/contributors"><img src="https://opencollective.com/typescript-eslint/contributors.svg?width=890" /></a>
+
+### Backers
+
+Thank you to all our backers! [[Become a backer](https://opencollective.com/typescript-eslint#backer)]
+
+<a href="https://opencollective.com/typescript-eslint#backers" target="_blank"><img src="https://opencollective.com/typescript-eslint/backers.svg?width=890"></a>
+
+### Sponsors
+
+Thank you to all our sponsors! (please ask your company to also support this open source project by [becoming a sponsor](https://opencollective.com/typescript-eslint#sponsor))
+
+<a href="https://opencollective.com/typescript-eslint/sponsor/0/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/0/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/1/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/1/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/2/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/2/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/3/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/3/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/4/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/4/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/5/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/5/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/6/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/6/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/7/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/7/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/8/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/8/avatar.svg"></a>
+<a href="https://opencollective.com/typescript-eslint/sponsor/9/website" target="_blank"><img src="https://opencollective.com/typescript-eslint/sponsor/9/avatar.svg"></a>
```

Comment:
I don't think contributors & sponsors information belongs in our `CONTRIBUTING.md`. We already showcase contributors on the root `README.md` and sponsors on the website. Let's keep each page as small as possible.

```suggestion
```

Reply:
cool